haddock-api/.hspec-failures

```diff
@@ -0,0 +1 @@
+FailureReport {failureReportSeed = 630888988, failureReportMaxSuccess = 100, failureReportMaxSize = 100, failureReportMaxDiscardRatio = 10, failureReportPaths = []}
```

Comment:
You can remove this file from the commit :)

Reply:
OK done.